tests/test_ostinato.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+import numpy.testing as npt
+import stumpy
+from stumpy.ostinato import _get_central_motif
+import pytest
+
+
+def naive_consensus_search(tss, m):
+    """
+    Brute force consensus motif from
+    <https://www.cs.ucr.edu/~eamonn/consensus_Motif_ICDM_Long_version.pdf>
+
+    See Table 1
+
+    Note that there is a bug in the pseudocode at line 8 where `i` should be `j`.
+    This implementation fixes it.
+    """
+    k = len(tss)
+
+    rad = np.inf
+    ts_ind = 0
+    ss_ind = 0
+
+    for j in range(k):
+        radii = np.zeros(len(tss[j]) - m + 1)
+        for i in range(k):
+            if i != j:
+                mp = stumpy.stump(tss[j], m, tss[i], ignore_trivial=False)
```

Comment:
In `tests/naive.py` you can import this into this test with `import naive` and then replace `stumpy.stump` with `naive.stump`

Reply:
This comment refers to the idea of `naive_get_central_motif`, right?

Reply:
No. Instead of using:

```
mp = stumpy.stump(tss[j], m, tss[i], ignore_trivial=False)
```

Please replace this with:

```
import naive

mp = naive.stump(tss[j], m, tss[i], ignore_trivial=False)
```

This `naive.stump` implementation is slower but helps us ensure that all of our fast implementations produce the same results as all of slow (naive) implementations.